tutorials/plot_background_filtering.py

```diff
@@ -785,6 +847,41 @@ def baseline_plot(x):
 # MNE-Python filtering functions with real data, consult the preprocessing
 # tutorial on :ref:`tut_artifacts_filter`.
 #
+# Defaults in MNE-C
+# -----------------
+# MNE-C by default uses:
+#
+#    1. 5 Hz transition band for low-pass filters.
+#    2. 3-sample transition band for high-pass filters.
+#    3. Filter length of 8197 samples.
+#
+# The filter is designed in the frequency domain, creating a linear-phase
+# filter such that the delay is compensated for as is done with the MNE-Python
+# ``phase='zero'`` filtering option.
+#
+# Squared-cosine ramps are used in the transition regions. Because these
+# are used in place of more gradual (e.g., linear) transitions,
+# a given transition width will result in more temporal ringing but also more
+# rapid attenuation than the same transition width in windowed FIR designs.
+#
+# The default filter length will generally have excellent attenuation
+# but long ringing for the sample rates typically encountered in M-EEG data
+# (e.g. 500-2000 Hz).
+#
+# Defaults in other software
+# --------------------------
+# A good but possibly outdated comparison of filtering in various software
+# packages is available in [7]_. Briefly:
+#
+# * EEGLAB
+#     MNE-Python in 0.14 defaults to behavior very similar to that of EEGLAB,
+#     see the `EEGLAB filtering FAQ`_ for more information.
+# * Fieldrip
+#     By default FieldTrip applies a forward-backward Butterworth IIR filter
+#     of order 4 (band-pass and band-stop filters) or 2 (for low-pass and
+#     high-pass filters). See e.g. `FieldTrip band-pass documentation`_
+#     for more information.
```

Comment:
I would add that this can be achieved in mne-python be adding `method='iir'` and `iir_params=dict(order=4)` to the `filter` method